pkg/console/operator/operator.go

```diff
@@ -101,6 +103,7 @@ func NewConsoleOperator(
 	// routes
 	routev1Client routeclientv1.RoutesGetter,
 	routeInformer routesinformersv1.RouteInformer,
+	oauthv1Client oauthclientv1.OAuthClientsGetter,
```

Comment:
wire an informer for OAuthClients here, derive a lister from it and add it to informers, as in https://github.com/openshift/console-operator/pull/808